BoardingPass/BoardingNavigationController.swift

```diff
@@ -91,43 +161,43 @@ public class BoardingNavigationController: UINavigationController {
      - parameter animated: Specify true to animate the transition or false
      if you do not want the transition to be animated.
      */
-    public func popToPreviousViewController(animated animated: Bool) {
-        if let poppableViewController = topViewController.flatMap(boardingInfoAfter) {
+    public func popToPreviousViewController(animated: Bool) {
```

Comment:
#req add isAnimated to make this line up with the `pishToNextViewController` function